front/app/components/IdeaCards/shared/Filters/SortingBox/index.tsx

```diff
@@ -31,15 +29,7 @@ const SortingBox = ({ handleSortOnChange, phaseId }: SortingBoxProps) => {
 
   return (
     <InputFilterCollapsible title={formatMessage(messages.sortBy)}>
-      <Box
-        bgColor={colors.white}
-        display="flex"
-        flexDirection="column"
-        p="12px"
-        pt="0"
-        borderRadius={theme.borderRadius}
-        gap="4px"
```

Comment:
Styles that applied to all collapsible filters were moved to InputFilterCollapsible as indicated above.